Doc/library/socket.rst

```diff
@@ -1079,6 +1079,14 @@ to sockets.
    to decode C structures encoded as byte strings).
 
 
+.. method:: socket.getblocking()
+
+   Return ``True`` if socket is in blocking mode, ``False`` if in
+   non-blocking.
```

Comment:
Victor: "getblocking() should be a simple as: return True if the timeout is equal to zero, False otherwise."

Yury: "That's exactly what is in the PR."

Hum, this doc doesn't mention the timeout but refers to a "blocking mode" which is not well defined for timeout > 0.

The socket module mentions 3 modes: blocking, non-blocking and "timeout mode":
https://docs.python.org/dev/library/socket.html#notes-on-socket-timeouts

Replace "if socket is in blocking mode" with "if the timeout is equal to zero", or replace "is in blocking mode" with "is in blocking or timeout mode"?

By the way, there is already a note: "At the operating system level, sockets in timeout mode are internally set in non-blocking mode."

Reply:
I think the docstring here should say "This is equivalent to `gettimeout() == 0`", just like the docstring for `setblocking` explains that it's equivalent to a certain `settimeout` call.